### ISSUE TITLE
indytube: No section: 'mencoder'

### ISSUE THREAD

**Issue by mgogoulos** [plumi/plumi.app#1117]
**Reported by denis on 29 Oct 2008 22:07 UTC**
I am testing plumi-0.2.1-final on debian lenny.

The transcodage fails, so i just try to launch indytube.py on command-line and have this error.

== python indytube.py               ==

```
Traceback (most recent call last):
  File "indytube.py", line 233, in <module>
    main()
  File "indytube.py", line 223, in main
    indytuber.parse_config(indyconf_path)
  File "indytube.py", line 26, in parse_config
    self.MENCODER_LOCATION=config.get('mencoder','MENCODER_LOCATION')
  File "/usr/lib/python2.5/ConfigParser.py", line 292, in get
    raise NoSectionError(section)
ConfigParser.NoSectionError: No section: 'mencoder'

```

**Comment by mgogoulos** [comment]
**Comment by andycat on 29 Oct 2008 22:34 UTC**
sounds like your indytube.conf isnt correct. can u paste your indytube.conf and the command line you are using to launch indytube.

**Comment by mgogoulos** [comment]
**Modified by andycat on 30 Oct 2008 02:29 UTC**

**Comment by mgogoulos** [comment]
**Comment by denis on 30 Oct 2008 09:59 UTC**
I have launched 
== python indytube.py ==

Here is my indytube.conf
```
[-of lavf -oac mp3lame -lameopts abr:br=64 -ovc lavc -lavcopts vcodec=flv:keyint=25:vbitrate=260:mbd=2:mv0:trell:v4mv:cbp:last_pred=3 -vf harddup,expand=:::::4/3,scale=320:240  -srate 22050 -ofps 25 -mc 0/10

[ffmpeg2theora](encoder]
BE_HOW_NICE=18
CONVERT_THESE=['.avi','.qt','.mov','.mpeg','.mp4','.mpg','.asf','.wmv','.3gp','.ogg']
DO_ENCODING=True
NUMBER_OF_PARALLEL_ENCODERS=1
ENCODER_LOCKFILE_BASE=/opt/indytube/plumidemo-encoder.lock
POLLTIME=60

[mencoder]
MENCODER_LOCATION=/usr/bin/mencoder
MENCODER_OPTIONS=-really-quiet)
CORTADO_LOCATION=http://plumi.domainepublic.net/flowplayer/cortado-ov-strippe.jar
FFMPEG2THEORA_COMMAND=ffmpeg2theora

[flvtool2]
FLVTOOL_LOCATION=/usr/bin/flvtool2

[paths]

VIDEO_FILE_DIRECTORY=/opt/Plone-2.5.5/zeocluster/client1/var/files/Members
FLV_FILE_DIRECTORY=/var/www/plumi-flv-files/
INCLUDE_FILE_DIRECTORY=/opt/Plone-2.5.5/zeocluster/client1/var/files/Members
INCLUDE_FILE_SUFFIX=.flv.inc
INCLUDE_TEMPLATE=/opt/indytube/template.include

[urls]
FLOWPLAYER_LOCATION=http://flv.domainepublic.net/flowplayer/FlowPlayerClassic.swf
VIDEO_SERVER_URL=http://flv.domainepublic.net/
SPLASH_IMAGE_BASE=http://plumi.domainepublic.net/flowplayer/
SPLASH_IMAGE_FILE=indytube.jpg

[logging]
LOG_FILE=/opt/indytube/plumi-wetube.log
LOG_LEVEL=logging.INFO

```

**Comment by mgogoulos** [comment]
**Comment by denis on 31 Oct 2008 20:34 UTC**
I just reinstall debian etch and the problem is the same

**Comment by mgogoulos** [comment]
**Comment by denis on 1 Nov 2008 18:19 UTC**
Just reinstall plumi-0-2-final in place of 0.2.1 and it's working now.

**Comment by mgogoulos** [comment]
**Comment by andycat on 3 Nov 2008 04:19 UTC**
This should fix it, it will go into plumi-0.2.2 

http://plumi.org/changeset/367/indytube

**Comment by mgogoulos** [comment]
**Modified by andycat on 18 Nov 2008 15:52 UTC**
